pom.xml

```diff
@@ -821,6 +821,13 @@
       </modules>
     </profile>
 
+    <profile>
```

Comment:
Could you just add java module directly instead of using profile ? We plan to remove all these interpreter specific profiles.